app/script/properties/PropertiesRepository.js

```diff
@@ -34,6 +39,7 @@ class PropertiesRepository {
 
     this.properties = new z.properties.PropertiesEntity();
     this.selfUser = ko.observable();
+    this.enableReadReceipts = ko.observable(PropertiesRepository.CONFIG.ENABLE_READ_RECEIPTS.defaultValue);
```

Comment:
Let's use the `propertiesEntity` that is responsible for holding all the user's properties